batch/extension-jberet/src/main/java/org/wildfly/extension/batch/jberet/BatchSubsystemDefinition.java

```diff
@@ -84,7 +84,7 @@
     static final SimpleAttributeDefinition DEFAULT_THREAD_POOL = SimpleAttributeDefinitionBuilder.create("default-thread-pool", ModelType.STRING, true)
             .setAllowExpression(false)
             .setAttributeGroup("environment")
-            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE)
+            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE) //todo this looks like it is missing capability reference
```

Comment:
The JavaDoc says:
> Records that this attribute's value represents a reference to an instance of a dynamic capability.

The `thread-pool` resource does not have a capability so I don't see how one could be referenced here. So while the `DEFAULT_THREAD_POOL` is a capability it doesn't not reference another capability.